projects/fishtornado_xr/Shark.cpp

```diff
@@ -78,7 +78,7 @@ void Shark::Shutdown()
     mMaterialConstants.Destroy();
 }
 
-void Shark::Update(uint32_t frameIndex)
+void Shark::Update()
```

Comment:
I think we can keep this as a single `Update` function and just call in the existing place.